tensorpack/dataflow/imgaug/geometry.py

```diff
@@ -43,17 +72,21 @@ class RotationAndCropValid(ImageAugmentor):
         Note that this will produce images of different shapes.
     """
 
-    def __init__(self, max_deg, interp=cv2.INTER_LINEAR):
+    def __init__(self, max_deg, interp=cv2.INTER_LINEAR, step_deg=0):
```

Comment:
If Rotation keeps the size when rotating 90 degree then this augmentor will be different (it doesn't keep the size).